front/src/modules/object-record/record-table/components/RecordTable.tsx

```diff
@@ -90,9 +125,34 @@ export const RecordTable = ({
 }: RecordTableProps) => {
   const tableBodyRef = useRef<HTMLDivElement>(null);
 
-  const { resetTableRowSelection, setRowSelectedState } = useRecordTable({
+  const tableRowIds = useRecoilValue(tableRowIdsState);
+
+  const {
+    scopeId: objectNamePlural,
+    resetTableRowSelection,
+    setRowSelectedState,
+  } = useRecordTable({
     recordTableScopeId: recordTableId,
   });
+
+  const { objectNameSingular } = useObjectNameSingularFromPlural({
+    objectNamePlural,
+  });
+
+  const { objectMetadataItem: foundObjectMetadataItem } = useObjectMetadataItem(
+    {
+      objectNameSingular,
+    },
+  );
+
+  const { createOneRecord: createOneObject } = useCreateOneRecord({
+    objectNameSingular,
+  });
+
+  const handleAddButtonClick = async () => {
+    await createOneObject?.({});
+  };
```

Comment:
The `createRecord` prop already does this, let's use the prop instead!

Reply:
Done